packages/@okta/vuepress-theme-default/assets/css/okta/pages/code.scss

```diff
@@ -8,3 +8,16 @@ ul.code-list {
   padding-left: 0px;
 }
 
+
+.language-content a:not([class="header-anchor header-link"]) {
+  display: block;
+}
+
+.language-content ul:not([class="language-tabs"]) {
```

Comment:
Same comment about `:not` here.

Is there a way we can get around using `!important` below, or are these styles mainly "legacy" CSS?

Reply:
@edburyenegren-okta I would love to do this, but for now, I can not wrap arbitrary elements through Markdown. I am looking at solution to be able to do this, but for now, using :not in the css is necessary to not include the deep link anchor for headers. .language-content is only used on code pages, so it will not affect anything else.

I will see if we can remove `!important` from this.